stack/rest/src/test/java/org/apache/usergrid/rest/filters/ContentTypeResourceIT.java

```diff
@@ -105,106 +75,84 @@ public void correctHeaders() throws Exception {
      */
     @Test
     public void textPlainContentType() throws Exception {
-        Map<String, String> data = hashMap( "name", "Solitaire2" );
-
-        String json = JsonUtils.mapToFormattedJsonString( data );
-
-        DefaultHttpClient client = new DefaultHttpClient();
-
-        HttpHost host = new HttpHost( super.getBaseURI().getHost(), super.getBaseURI().getPort() );
+        refreshIndex();
+        String json = JsonUtils.mapToFormattedJsonString(hashMap("name", "Solitaire2"));
+        WebResource.Builder builder = app().collection("games").getResource(true, getAdminToken())
+            .accept(MediaType.APPLICATION_JSON)
+            .type(MediaType.APPLICATION_JSON);
 
-        HttpPost post = new HttpPost( String.format( "/%s/%s/games", context.getOrgUuid(), context.getAppUuid() ) );
-        post.setEntity( new StringEntity( json ) );
-        post.setHeader( HttpHeaders.AUTHORIZATION, "Bearer " + context.getActiveUser().getToken() );
-        post.setHeader( HttpHeaders.ACCEPT, MediaType.APPLICATION_JSON );
-        post.setHeader( HttpHeaders.CONTENT_TYPE, MediaType.TEXT_PLAIN );
+        ClientResponse clientResponse = builder.post(ClientResponse.class, json);
```

Comment:
Is there a reason this couldn't use the POST command you specified in the named resource?